EIPS/eip-8032.md

```diff
@@ -105,8 +197,6 @@ TODO
 
 Needs discussion.
 
-A remark was made that, were many contracts updated to include a `depth` field in a single block, this would represent writes to about 3k accounts. This could put stress on client databases. Since this change only applies to large contracts, the risk of this happening is already mitigated by the cost of filling the storage of such contracts.
-
```

Comment:
I suggest doing the reverse and explaining why we're not doing the depth after all:

we considered using the depth as a proxy, however even large contracts have a depth that is very cheap to find a key collision for. While this approach would save the need for a conversion step, even the largest contracts aren't deep enough for a depth-attack to be infeasible.